org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/DeviceResponse.java

```diff
@@ -156,21 +160,21 @@ private static void processScannables(DeviceRequest request, IScannableDeviceSer
 	private static void merge(DeviceInformation<?> info, IScannable<?> device) throws Exception {
 		info.setLevel(device.getLevel());
 		info.setUnit(device.getUnit());
-        info.setUpper(device.getMaximum());	
+        info.setUpper(device.getMaximum());
         info.setLower(device.getMinimum());
         info.setPermittedValues(device.getPermittedValues());
         info.setActivated(device.isActivated());
         info.setMonitorRole(device.getMonitorRole());
  	}
 
 	private static void processRunnables(DeviceRequest request, IRunnableDeviceService dservice) throws Exception {
-		
+
 		if (request.getDeviceName()!=null) { // Named device required
 			IRunnableDevice<Object> device = dservice.getRunnableDevice(request.getDeviceName());
 			if (device==null) throw new EventException("There is no created device called '"+request.getDeviceName()+"'");
-			
+
 			// TODO We should have a much more reflection based way of

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)